### ISSUE TITLE
How to adjust whitebalance for NOIR camera module?

### ISSUE THREAD

**Issue by starlingfire** [kingkingyyk/Pi-MJPEG-Server#2]
First of all, this is amazing. Super fast, quality network camera to replace motioneyeos on my Pi 4 with the v3 camera module. Love it!

However, I'm using the NOIR version of the camera module, so the colors are off. In motioneye, I would add the motion option `mmalcam_control_params -awb greyworld` to fix this. Is there a way to accomplish this with your script?

**Comment by kingkingyyk** [comment]
Hi @starlingfire, I did some search in Pi forum and found out someone used [tuning file](https://forums.raspberrypi.com/viewtopic.php?t=347194#p2079782) for libcamera to achieve the same effect.

Hence, I have added the support of specifying tuning file.
https://github.com/kingkingyyk/Pi-MJPEG-Server/blob/master/mjpeg_server.py#L26

As for the tuning file path, it seems to be changed in new release.
Older Raspbian : `/usr/share/libcamera/ipa/raspberrypi/`
Latest Raspbian :  `/usr/share/libcamera/ipa/rpi/vc4/`

In the directory you can find the preset for list of cameras. For V3 NoIR, it would be `imx477_noir.json`.

In short, pull the latest master to your Pi, then set the `TUNING_FILE` value to `/usr/share/libcamera/ipa/raspberrypi/imx477_noir.json` or `/usr/share/libcamera/ipa/rpi/vc4/imx477_noir.json`.

I tested applying `imx219_noir.json` on my V2 module and the color looked better than the default preset. 🤣
